src/plugins/intel_gpu/tests/unit/test_cases/gemm_gpu_test.cpp

```diff
@@ -1739,6 +1739,52 @@ TEST(gemm_onednn, impl_replacement_with_cldnn) {
     ASSERT_FALSE(impl->is_dynamic());
 }
 
+TEST(gemm_onednn, input0_simple_input1_fsv16_select_ref_kernel) {
+    auto& engine = get_test_engine();
+    if (!engine.get_device_info().supports_immad)
+        return;
+
+    tests::random_generator rg(GET_SUITE_NAME);
+    int input_b = 1, input_f = 16, input_y = 3, input_x = 3;
+    int output_f = 16;
+
+    auto input_size = tensor(input_b, input_f, input_x, input_y);
+    auto input_data = rg.generate_random_4d<ov::float16>(input_b, input_f, input_y, input_x, -1, 1);
```

Comment:
Layout is enough when check with only model build?

Reply:
My answer is yes.
The main purpose of the unittest is happening exception for given condition (input0:simple, input1:non-simple, output:simple) before this PR. And this PR fixes the issue during build.